src/Calculator/Views/CalculatorProgrammerRadixOperators.xaml.cpp

```diff
@@ -47,7 +47,12 @@ void CalculatorProgrammerRadixOperators::FlyoutButton_Clicked(_In_ Platform::Obj
 
 void CalculatorProgrammerRadixOperators::checkDefaultBitShift()
```

Comment:
In previous versions, there was an issue where button availability was reset when opening BitShift Flyout not when the mode is changed